PRChecker/Shared/Models/PullRequest.swift

```diff
@@ -7,21 +7,99 @@
 
 import Foundation
 import Apollo
+import SwiftUI
 
-enum PRState {
-    case open
-    case merged
+enum PRState: String {
+    case open = "Open"
+    case merged = "Merged"
```

Comment:
Is there a state named `Closed` in response?

Reply:
There can be. Right now I only query Github for Open/Merged, but Closed is the last option.